### PR TITLE
Bug 1122328 - Add Finnish (fi), Armenian (hy-AM), Ewe (ee), Lingala (ln)...

### PR DESCRIPTION
**Pull request by TheoChevalier**
... and Wolof (wo), Igbo (ig), Yoruba (yo), Hausa (ha) and Burmese (my) locales, add Wolof keyboard + generic keyboard for Africa

### CODE REVIEW

**Comment by try-server-hook** [comment]
[TheoChevalier Théo Chevalier (TheoChevalier)](https://github.com/TheoChevalier) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=56bc6ae724df)

**Comment by try-server-hook** [comment]
[TheoChevalier Théo Chevalier (TheoChevalier)](https://github.com/TheoChevalier) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1f05064a966a)

File: apps/keyboard/js/layouts/en-Africa.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      { value: '⇪', ratio: 1.5, keyCode: KeyEvent.DOM_VK_CAPS_LOCK },
+      { value: 'z' }, { value: 'x' }, { value: 'c' }, { value: 'v' },
+      { value: 'b' }, { value: 'n' }, { value: 'm' },
+      { value: '⌫', ratio: 1.5, keyCode: KeyEvent.DOM_VK_BACK_SPACE }
+    ], [
+      { value: '&nbsp', ratio: 8, keyCode: KeyboardEvent.DOM_VK_SPACE },
+      { value: '↵', ratio: 2, keyCode: KeyEvent.DOM_VK_RETURN }
+    ]
+  ],
+  pages: [undefined, {
```
Comment (RudyLu) [inline apps/keyboard/js/layouts/en-Africa.js]: Please help define the first page here, see en-GB.js as an example.

File: apps/keyboard/js/layouts/wo.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      { value: 't' }, { value: 'y' }, { value: 'u' }, { value: 'i' },
+      { value: 'o' }, { value: 'p' }
+    ], [
+      { value: 'q' }, { value: 's' }, { value: 'd' }, { value: 'f' },
+      { value: 'g' }, { value: 'ë' }, { value: 'j' }, { value: 'k' },
+      { value: 'l' }, { value: 'm' }
+    ], [
+      { value: '⇪', ratio: 1.5, keyCode: KeyEvent.DOM_VK_CAPS_LOCK },
+      { value: 'w' }, { value: 'x' }, { value: 'c' }, { value: 'ŋ' },
+      { value: 'b' }, { value: 'n' }, { value: "'", keyCode: 39 },
```
Comment (RudyLu) [inline apps/keyboard/js/layouts/wo.js]: Please remove the keyCode for `'` and change it to `{ value: '\''}`

**Comment by try-server-hook** [comment]
[TheoChevalier Théo Chevalier (TheoChevalier)](https://github.com/TheoChevalier) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dcb4448151f1)

File: apps/keyboard/js/layouts/wo.js (hunk 2 above)
Comment (TheoChevalier) [inline apps/keyboard/js/layouts/wo.js]: I assume this change is for all branches, right (including 2.1/2.0)?

**Pull request merged**